www/js/controllers.js

```diff
@@ -301,7 +303,7 @@ angular.module('app.controllers', ['app.services','ngLodash','truncate','ngIOS9U
             console.log(success);
             localStorageService.set('fullName',success.name);
             $rootScope.user.fullName = success.name;
-
+            $rootScope.gender = success.gender;
```

Comment:
@wal-nadeem  Have you verified that the gender is actually getting passed in this object? It wasn't before.